components/prism-css-extras.js

```diff
@@ -55,7 +55,9 @@
 					lookbehind: true
 				}
 			],
-			'punctuation': /[()]/
+			'combinator': />|\+|~|(?:\|\|)]/,
```

Comment:
Please add a test for all combinators.
Also, note the `]`
```suggestion
			'combinator': />|\+|~|\|\|/,
```

Reply:
Yes, your code is better than mine.

The `]` will be removed.